app/helpers/trainee/subjects_helper.rb

```diff
@@ -0,0 +1,36 @@
+module Trainee::SubjectsHelper
+  def count_user_tasks_done user, tasks
+    UserTask.by_user(user)&.tasks_done&.by_task(tasks)
```

Comment:
```suggestion
  def user_tasks_done user, tasks
    UserTask.by_user(user).tasks_done.by_task(tasks)
```
scope nối k cần toán tử an toàn